react-common/components/profile/Profile.tsx

```diff
@@ -44,5 +46,24 @@ export const Profile = (props: ProfileProps) => {
             userState={userBadges}
             onBadgeClick={onBadgeClick}
         />}
+
+        {profileSmall &&
+            <div className="profile-info-container">
+                <p className="profile-info">
+                    {lf("Now that you're logged in, your projects will be automatically saved to the cloud so you can access them from any device! ")}
+                    {lf("Learn more at ")}
+                    <a href="https://arcade.makecode.com/identity/cloud-sync" target="_blank" rel="noopener noreferrer" tabIndex={0}>{lf("Cloud Sync ")}</a>
+                    {lf("or ")}
+                    <a href="https://makecode.com/privacy-faq" target="_blank" rel="noopener noreferrer" tabIndex={0}>{lf("Privacy ")}</a>
+                    {lf("or ")}
+                    <a href="https://arcade.makecode.com/identity/sign-in" target="_blank" rel="noopener noreferrer" tabIndex={0}>{lf("Identity.")}</a>
```

Comment:
Can we just make a single docs page at `/identity` that links out to whatever else we want here? having a bunch of links under an or feels a little messy to me. Probably should go in common-docs/ so it's available per target

Reply:
Oh, I actually just realized that when you sign-in, you can hit the `identity/sign-in` page with the learn more link in the lower right hand corner of the modal. And with the privacy link everywhere, I don't think that's necessary in this pop-up either. If we were going to have any link here, I would consider https://arcade.makecode.com/identity/cloud-sync, but I would make an argument against the other two.